compute_tools/src/compute_prewarm.rs

```diff
@@ -148,6 +111,7 @@ impl ComputeNode {
         fail::fail_point!("compute-prewarm", |_| bail!("compute-prewarm failpoint"));
 
         info!(%url, "requesting LFC state from endpoint storage");
+        let mut now = Instant::now();
```

Comment:
Such things look like a candidate for a good RAII guard :-)

Reply:
I'm not aware of any which are time-related